packages/generator/ni_measurement_plugin_sdk_generator/client/templates/measurement_plugin_client.py.mako

```diff
@@ -43,7 +43,7 @@ class Output(NamedTuple):
 % endif
 
 
-class ${class_name}:
+class ${repr(class_name)}:
```

Comment:
Doesn't repr include the quotes? That shouldn't work: `class "MyClass":`

For class_name I think it's more appropriate to call isidentifier() and return an error if the class name is not a valid identifier. Maybe this is already covered?

Reply:
Yes, we have validated with `isidentifier()` already. So I am removing the `repr` here.